go/engine/signup_test.go

```diff
@@ -321,3 +322,58 @@ func TestSignupNonAsciiDeviceName(t *testing.T) {
 		require.IsType(t, err, testVal.err)
 	}
 }
+
+func TestSignupNOPWBadParams(t *testing.T) {
+	tc := SetupEngineTest(t, "signup_nopw")
+	defer tc.Cleanup()
+
+	fu, _ := NewFakeUser("sup")
+	arg := MakeTestSignupEngineRunArg(fu)
+	arg.StoreSecret = false
```

Comment:
For the record, I believe the `StoreSecret` field is from ancient times where we allowed it to be off.  It should always be true.